spring-integration-ip/src/main/resources/org/springframework/integration/ip/config/spring-integration-ip-4.3.xsd

```diff
@@ -80,8 +80,29 @@
 							</xsd:appinfo>
 						</xsd:annotation>
 					</xsd:attribute>
-					<xsd:attribute name="host" type="xsd:string" />
+					<xsd:attribute name="host" type="xsd:string"/>
+					<xsd:attribute name="destination-expression" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								A SpEL expression to determine the destination address for the datagram packet
+								at run time. The 'requestMessage' is used as a root object for evaluation context.
```

Comment:
`Runtime` is a word to identify the time of execution of our application: https://en.wikipedia.org/wiki/Runtime_system.
So, if your IDE suggests to change it, just let it know that we are correct.
Save the word to the Typo plugin! :smile: